### PR TITLE
Small update in readme and remove default .env file (exclude from git)

### PR DESCRIPTION
**Pull request by gigimon**
I have removed doc and list runner loggers because it duplicates all data. Before:
```
 ✔ Contract: ERC2771Context when called directly msgSender returns the transaction sender when from another contract: 1448ms
                <dt>returns the transaction sender when from another contract</dt>
                <dt>/Users/gigimon/workspaces/neon/neon-compatibility/openzeppelin-contracts/test/metatx/ERC2771Context.test.js</dt>
                <dd><pre><code>async function () {
      const { tx } = await this.caller.callSender(this.context.address, { from: sender });
      await expectEvent.inTransaction(tx, ContextMock, &#x27;Sender&#x27;, { sender: this.caller.address });
    }</code></pre></dd>
              </dl>
            </section>
      msgData
            <section class="suite">
              <h1>msgData</h1>
              <dl>
    Contract: ERC2771Context when called directly msgData returns the transaction data when called from an EOA:         ✔ returns the transaction data when called from an EOA (1292ms)
```

Now:
image

I removed this because:
1. list and spec duplicate each other
2. doc - post html code for page

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -6,4 +6,4 @@ ADDRESS_TO=
 DISABLE_CONFIRMATION=false
 PRIVATE_KEY=
 FAUCET_QUOTIENT=10
-FAUCET_URL=https://proxy.teststand.neontest.xyz/request_erc20_tokens
+FAUCET_URL=https://proxy.teststand.neontest.xyz/request_erc20_tokens
```
Comment (apetrovskiy) [inline .env]: Just curious, why did you remove the ending empty line? It's like a standard in many styles to have an empty line in the end of file

**Review by apetrovskiy — APPROVED** [review]

**Comment by gigimon** [comment]
Logging before https://github.com/neonlabsorg/neon-compatibility/runs/4206273812?check_suite_focus=true:
Merge pull request #105 from apetrovskiy:feature:103-more-data-about-environment · neonlabsorg:neon-compatibility@3e64217 2021-11-15 12-26-06
Merge pull request #105 from apetrovskiy:feature:103-more-data-about-environment · neonlabsorg:neon-compatibility@3e64217 2021-11-15 12-24-57

Logging after https://github.com/gigimon/neon-compatibility/runs/4189175507?check_suite_focus=true:
Update reporterConfig json · gigimon:neon-compatibility@7c4e3fa 2021-11-15 12-21-30
Update reporterConfig json · gigimon:neon-compatibility@7c4e3fa 2021-11-15 12-21-03
